astroid/astroid_manager.py

```diff
@@ -1,5 +1,7 @@
 """
-This file contain the global astroid MANAGER, to prevent circular import that happened
+This file contain the global astroid MANAGER, to prevent circular import that
+happened.
```

Comment:
This sentence is now broken.

Reply:
I missed so much of those that I'll need to redo the commit entirely. I'll get to it. There's a lot of change to review.